### PR TITLE
show release version when listing clusters

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2760. This is successfully tested on `gauss`. 

```
$ gsctl list clusters
ID     ORGANIZATION  NAME                     RELEASE  CREATED
omq8y  giantswarm    Timo: first update test  3.1.1    2018 Mar 07, 13:45 UTC
```

### CODE REVIEW

File: commands/list_clusters.go [hunk 1]
```diff
@@ -124,6 +124,7 @@ func clustersTable(args listClustersArguments) (string, error) {
 		color.CyanString("ID"),
 		color.CyanString("ORGANIZATION"),
 		color.CyanString("NAME"),
+		color.CyanString("RELEASE"),
```
Comment (oponder) [inline commands/list_clusters.go]: Should we call this `RELEASE VERSION` as well @marian?

Comment (marians) [reply commands/list_clusters.go]: II am fine with the brief version here.

Comment (xh3b4sd) [reply commands/list_clusters.go]: Then it would be nice to get an approval on this.

**Comment by marians** [comment]
For clusters that don't have a `release_version` (or rather: have an empty string) we should display the `n/a` placeholder. This can be tested with ginger:

```
ID     ORGANIZATION  NAME                 RELEASE  CREATED
ohi6z  giantswarm    Ross ELB blogpost    3.1.0    2018 Mar 07, 16:58 UTC
vxvc7  giantswarm    Giantswarm Frontend           2017 Aug 29, 04:26 UTC
```

**Comment by xh3b4sd** [comment]
Good catch. Sorted. 

```
$ gsctl list clusters
ID     ORGANIZATION  NAME                 RELEASE  CREATED
ohi6z  giantswarm    Ross ELB blogpost    3.1.0    2018 Mar 07, 16:58 UTC
vxvc7  giantswarm    Giantswarm Frontend  n/a      2017 Aug 29, 04:26 UTC
```

**Pull request merged**